content/docs/how-we-work/security.md

```diff
@@ -10,6 +10,29 @@ Security is an everlasting concern in the world of software engineering. Underst
 
 That said, security is a wide and deep topic and a single person cannot hope to span the width and depth of this challenging area. It is the team's responsibility to understand security at various levels and implement safeguards.
 
-For example, an application relies on database credentials to connect to the database. There can be API keys using which the application communicates with the third-party service. They must not be kept in the repository. There should not even be a Git commit referring to those credentials. They can be misused if a malicious user gets hold of them.
+For example, an application relies on database credentials to connect to the database. There can be API keys using which the application communicates with the third-party service. They must not be kept in the repository. There should not even be a Git commit referring to those credentials. They can be misused if a malicious user gets hold of them. Use the environment variables or vaults for storing and retrieving them.
```

Comment:
```suggestion
For example, an application relies on database credentials to connect to the database. There can be API keys using which the application communicates with the third-party service. They must not be kept in the repository. There should not even be a Git commit referring to those credentials. They can be misused if a malicious user gets hold of them. Use environment variables or a vault for storing and retrieving them.
```

Please consider running the changes through Grammarly.

Reply:
Noted. Though that content was already on EHB.

Reply:
I'm aware. We should still aim to fix issues that we see. We don't have to do that exhaustively right now but if a line is changed, we can certainly fix any issues that we see.